.github/workflows/build-firmware.yml

```
@@ -4,314 +4,57 @@ on:
+          { id: cyd-2432s022c, arch: esp32 },
+          { id: cyd-2432s024c, arch: esp32 },
+          { id: cyd-2432s028r, arch: esp32 },
+          { id: cyd-2432s028rv3, arch: esp32 },
+          { id: cyd-2432s032c, arch: esp32 },
+          { id: cyd-4848s040c, arch: esp32s3 },
+          { id: cyd-8048s043c, arch: esp32s3 },
+          { id: cyd-e32r28t, arch: esp32 },
+          { id: cyd-e32r32p, arch: esp32 },
+          { id: cyd-e32r35t, arch: esp32 },
+          { id: cyd-e32r40t, arch: esp32 },
+          { id: cyd-2432s032c, arch: esp32 },
```

actionlint (1.7.8)

duplicate value {"arch": "esp32", "id": "cyd-2432s032c"} is found in matrix "board". the same value is at line:20,col:11

(matrix)